projects/components/src/color-picker/color-picker.component.ts

```diff
@@ -94,6 +106,10 @@ export class ColorPickerComponent implements ControlValueAccessor, OnChanges {
     this.propagateControlValueChangeOnTouch = onTouch;
   }
 
+  public setDisabledState(isDisabled?: boolean): void {
```

Comment:
This is fine, but could also do `public setDisabledState(isDisabled: boolean = false): void {`

And remove the `??` check below.

Reply:
careful - if these are called by angular and it sends in nulls, the current code would assign false and the suggestion would assign null. May not be relevant, but something to keep in mind when working in code that interfaces with angular.

Reply:
I prefer to keep this change to keep the consistency with the rest of CVA components.

Reply:
Is there a concern for angular or other code we don't own calling this method? If so, my suggestion above is safer.

Reply:
this method is from the CVA interface, angular forms calls it. I'm not sure why we default it at all thoguh, that's not part of angular's contract:
https://angular.io/api/forms/ControlValueAccessor#setdisabledstate

Reply:
I took a look at the documentation and it's no concern. I just tried to keep the consistency with the rest of the components but already changed.